### PR TITLE
add deno permission

### PR DESCRIPTION
**Pull request by gonzalovelasco**

### CODE REVIEW

File: .github/workflows/deno_deploy.yml [hunk 1]
```diff
@@ -18,10 +18,10 @@ jobs:
       - uses: denoland/setup-deno@v1
         with:
           deno-version: v1.x
-      
+
       - name: Generate manifest file for dynamic imports
         run: |
-          deno run --allow-read --allow-write routes-manifest-generate.ts
+          deno run --allow-read --allow-write --allow-net routes-manifest-generate.ts
```
Comment (matthewwong525) [inline .github/workflows/deno_deploy.yml:24]: this is used to generate the routes-manifest might not help with the deploying itself

**Review by matthewwong525 — APPROVED** [review]
give it a shot and see if it works

File: plugins/official/speech-to-text/index.ts [hunk 2]
```diff
@@ -10,7 +10,6 @@ export default async (request: Request): Promise<Response> => {
   }
   try {
     const { fullPath } = await download(source);
-    console.log(fullPath);
```
Comment (matthewwong525) [inline plugins/official/speech-to-text/index.ts:13]: good job

File: .github/workflows/deno_deploy.yml (hunk 1 above)
Comment (gonzalovelasco) [reply .github/workflows/deno_deploy.yml:24]: It's in the library documentation.

**Pull request merged**